---
title: "Lab Manual Conservation Biology Fall 2020"
subtitle: "Conservation Biology Fall 2020"
author: "Shannon J. O'Leary"
date: "`r Sys.Date()`"
knit: "bookdown::preview_chapter"
output:
  msmbstyle::msmb_html_book:
    highlight: tango
    toc: TRUE
    toc_depth: 1
    split_by: chapter
    margin_references: FALSE
    css: msmb.css
bibliography: labmanual.bib
link-citations: yes
editor_options: 
  chunk_output_type: console
---

```{r include=FALSE}

# load libraries
library(msmbstyle)
library(knitr)

# global settings for code chunks
knitr::opts_chunk$set(tidy = FALSE, 
                      message=FALSE, 
                      warning=FALSE,
                      cache.extra = packageVersion('msmbstyle'))

# invalidate cache when the tufte version changes
options(htmltools.dir.version = FALSE)

```

# Shark fin trade monitoring

**Date**: 10/14/2020

Download the project folder [here](https://drive.google.com/drive/folders/15byL9emkVrO6v1rZ2gXf1UJCinYFRvoN?usp=sharing). 

Make sure to download the **entire** folder (not just the contents). 

You can open the `Rproject` directly by double-clicking on the file which will launch `Rstudio`, or once you have opened `Rstudio` open the project folder using `File -> Open Project` or using the `Rproj` icon in the top right corner above the environment panel.


**Learning Objectives**

After completing this lab you should

* understand what genetic barcoding is.
* understand how genetic barcoding can be used as a tool for wildlife forensics.
* understand how to use `NCBI`s `Genbank` database and the `BLAST` to identify a species for an unknown COI sequence.
* understand how genetic barcoding can be used to determine the species composition of a shark fin market.
* understand how to use `fishbase` and other online resources to get basic information for the ecology, biology, conservation status, and CITES listing of a fish species.

<br>


## Molecular Ecology & Conservation Genetics

The central question of ecology is how organisms interact with each other and their environment and understanding the resulting observed patterns of distribution and abundance. The field of molecular ecology has arisen as a reaction to the limited ability to be able to observe some biological processes *in situ* and directly obtain data. Instead, molecular genetics techniques, population genetics theory and techniques from fields such as landscape genetics ecology, and bioformatics are applied to understand questions in ecology, evolution, behavior, and conservation biology. 

This toolkit includes a variety of approaches, both in terms of available genetic markers and the analytic methods used to analyze them. In general, methods fall into two broad categories.

(1) One of the central paradigms of molecular ecology is the assumption that we can use signals in genomic data sets to understand the ecology and demographic history of populations because evolutionary forces shape the patterns of within and between group patterns of genetic diversity - the specific application of these methods to small populations is studied in the field of **Conservation Genetics**.

(2) We can also use genetics to characterize and identify (groups of) individuals (DNA fingerprinting), populations (mixed stock analysis), and species (genetic barcoding). 

**Genetic barcoding** is a technique that is increasingly used for wildlife forensics because it allows identification of unknown and frequently degraded samples that cannot be morphologically identified. Under certain circumstances, it can also be used to identify the geographic region of origin. 

Today, we will learn how to use genetic markers (barcodes) to be able to identify unknown specimen using a variety of tissue and other samples types. We are going to apply this in the context of monitoring the trade of shark fins in the Guangzhou (mainland China) market. We will be using the raw data set from

**Cardeñosa et al. (2020). Species composition of the largest shark fin retail-market in mainland China. Scientific Reports, 10(1), 1-10.**

This is one of the most recent studies in a series of efforts using genetic barcoding to understand the species composition of the Hong Kong shark fin market:

* Clarke et al. (2006). Identification of shark species composition and proportion in the Hong Kong shark fin market based on molecular genetics and trade records. Conservation Biology, 20(1), 201-211.
* Clarke et al. (2005). Estimates of shark species composition and numbers associated with the shark fin trade based on Hong Kong auction data. Journal of Northwest Atlantic Fishery Science, 35, 453-465.
* Fields et al. (2018). Species composition of the international shark fin trade assessed through a retail‐market survey in Hong Kong. Conservation Biology, 32(2), 376-389.


## Genetic Barcoding

Genetic barcoding allows for identification of an unknown specimen for which DNA can be obtained using a short section of DNA using the same specific gene or set of genes across a very broad taxonomic group. 

To be able to do this you need a gene that is present in all (or a significantly large group of species), flanked by conserved regions across a wide range of species so that universal primers can be designed. Further, the gene must be variable among species but conserved within a species.

`r msmbstyle::question(label = "ques:one", text = "For Eukaryotes Cytochromeoxidase I has emerged as the barcoding gene. Explain how this gene fulfills all the requirements.")`

Next to an appropriate gene, you will also need a reference database, i.e. sequences for the barcoding gene from a set of vouchered specimen. That way, as long as you can extract DNA from a sample you can identify the species even if it is too degraded or "incomplete" for a morphological ID.

Barcoding is widely applied in the context of conservation biology, specifically to

* identify/stop illegal trading (ivory, shark fins)
* identify food fraud (proper labeling, fish/sushi)
* identify life stages otherwise hard difficult to distinguish (larval stage of insects, amphibians, fish)

In practice, genetic barcoding usually consists of these steps:

1. Obtain sample
2. Extract DNA
3. Perform PCR using universal primers to amplify barcoding gene (for animals usually COI)
4. Sequence amplified gene
5. Compare sequence to reference database to get sample ID

`r msmbstyle::question(label = "ques:two", text = "DNA can be extracted from fin clips, scales, fur, pieces of skin, scat, bones, ... this means that you can also use non-lethal sampling. In this light, explain how barcoding could be a useful tool for conservation in general, and specifically for wildlife forensics and enforcing regulations in the wildlife trade (For your argument, consider how the fact that you can use non-lethal sampling, obtain DNA from a wide array of samples is particularly useful and assess how complicated/straightforward the labwork/analysis is).")`


##  Shark Fins/fishery

Shark fin soup is considered a delicacy in Chinese culture and is frequently served as a luxury item and on special occasions, including weddings and banquets. While the taste itself comes from other ingredients, the shark fins mainly provide texture.

Increasing wealth of the middle class resulted in an increased demand for fins. As a result shark fin prices rapidly increased and the practice of shark fining became increasingly lucrative. Because the fins are so much more valuable than the meat of the fish, it became common practice for fins to be cut off of sharks and the carcasses to be tossed back into the water. Ethical concerns compounded with concerns over unsustainable shark fishing in general, resulted in public outcry and campaigns to ban shark fining and in the some countries there has been increasing pressure for restaurants to stop serving shark fin soup all together. 

In the late 90s and early 2000s, fisheries-driven declines of some shark species were linked to the fin trade. Despite the fin trade comprising a valuable global trade, monitoring it on a species-specific basis is difficult as few countries with commercial or subsistence shark fisheries keep accurate records or regulate specific shark fisheries. As a result, it can be difficult to quantify the impact of the fin trade on specific shark populations/species and resulting declines.

`r msmbstyle::question(label = "ques:three", text = "Based on sharks position in the trophic level, their habitat, geographic range(s), and life history, argue whether you think shark populations are intrinsically vulnerable to local extinctions due to increased fishing mortality.")`

Historically, mainland China is the world's second largest importer of shark fins after Hong Kong and primary consumer of shark fin soup. The fins used for shark fin soup are the cartilaginous dorsal, pectoral, and caudal fins. Most global shark catch and trade data is aggregated and due to the use of a variety of common names for (groups of sharks) it is difficult to trace products through supply chains and understand the pressure on shark populations due to fishing. Understanding the species composition of shark fin markets and monitoring the trade is becoming increasingly important, partially to regulate trade of CITES-listed sharks. Agency challenged with enforcing trade restrictions and uncovering illegal trade frequently cite difficulties for species identification of fins once they have been cut-off of the shark^[The argument is that all shark fins look the same - [or do they](https://www.identifyingsharkfins.org/)? Significant efforts have been made to develop dichotomous keys to allow for straightforward identification by law enforcement.]

`r msmbstyle::question(label = "ques:four", text = "Consider what type of information/samples you would need to identify illegally traded fins and species composition if you were to use traditional methods like morphologically identifications compared to genetic barcoding.")`

Because there is a general apprehension of importers and retailers to donate fins or provide other samples for genetic testing and buying fins at a sufficiently large scale is prohibitively expensive, there have been several studies that use trimmings produced during processing of fins at which time excess skin, muscle, and cartilage is removed. These trimmings are collected and sold for comparatively low prices.

`r msmbstyle::question(label = "ques:five", text = "Briefly discuss the advantages and limitations of using trimmings as proxy for fins sold in a market to determine species composition.")`

For this study, the shark fin retail market of Gangzhou was sampled ten times (2015 - 2017). To obtain a random and representative sample, first, a list of known vendors was generated. Then, a list of 10 vendors was randomly generated  for each sampling event and two bags of trimmings obtained. From each bag, ten trimmings were randomly selected, DNA was extracted and COI (barcoding gene) was amplified and sequenced.


## Using blast to identify DNA sequences

Your `data` folder contains a file called `shark_fins.fasta`. 

`fasta` files are a specific format for storing genetic information it consisting of a header line starting with `>` and followed by information like a sample name, species information ... to identify the sequence. This is followed be the DNA sequence itself in the subsequent line(s). 

A `multi-fasta` file like the one in your `data` folder contains multiple sequences, each sequence has its own header line with unique identifiers.

`Genbank` is a "comprehensive public database of nucleotide sequences and supporting bibliographical and biological annotation" and serves as a repository for sequence information generated for published (and some unpublished) scientific studies. Most journal require you to submit the sequence data you generate to a database, in the last few years some alternatives to `genbank` have arisen, though genbank still remains the most comprehensive to date. 

We will use the Basic Local Alignment Tool, [BLAST](https://blast.ncbi.nlm.nih.gov/Blast.cgi) to compare our sequences to COI sequences that have been deposited in `genbank`.

To do this go to the BLAST [website](https://blast.ncbi.nlm.nih.gov/Blast.cgi). Here, you can choose a BLAST program to run. We will be doing a nucleotide blast, so select that option. 

Now, open the `multi-fasta` file so you can take a look at the sequences. You can do this by clicking on it in the file navigation pane (bottom left), it will then open in your Viewer pane in Rstudio (top right). 

Chose a random sequence, highlight just the sequence information and copy it to your clipboard^[You can use `CTRL+C`/`Command+C` or `right click` and select `copy`.]. 

Now, in the browser window with `Blast` open, paste your sequence into the box `Enter Query Sequence`.

Next, select the database to be searched: `Nucleotide Collection (nr/nt)`. 

In general, your `Program Selection` will be based on the goal of your search. Here, you are attempting to identify an unknown sample – so we will use `highly similar sequences (megablast)`. Other options are more dissimilar sequences (`discontinuous megablast`) to be used to find closely related species and somewhat similar sequences (`blastn`) to be used for short (<20bp) searches.

Finally, click the `BLAST` button to initiate your query. A `BLAST` search is a pairwise comparison between the DNA sequence you have uploaded and all the DNA sequences deposited in `Genbank`; so this might take a few seconds. 

You will receive a BLAST report once the search is completed. It consists of a list of your search results detailing the Accession number, species ID, and scores & statistics ranking your results.

Your results are ranked by the **Max Score/Total score** which scores the pairwise comparison between your DNA sequence and the search results. Higher scores mean better alignments. Further, you will receive and **e-value** which indicates if the results would be expected by chance alone. The lower the `e-value` the less likely the results will be obtained by chance.

For more information on your query results select a result by clicking on the hyperlink.

The detailed description contains information on the Accession number (a unique identifier code for a sequence), a description of the species, BLAST scores & Percent identity and a Pairwise Alignment of your query DNA sequence and the result. 

For even more information on the sequence it has been matched to, you can click on the accession number.

The pairwise alignment compares your query (unknown) sequence to the subject (result) sequence. Your query will generally begin at nucleotide 1, though the best matching subject sequence might not. 

`r msmbstyle::question(label = "ques:six", text = "Select a set of five random sequences from the fasta file and perform a blast. For each record the identified species, e-value, and percent match for the five best matches.")`

Don't worry, you will not have to perform a blast for every species in the `fasta` file. I have used a more efficient method using a local blast data base and a command line interface to get our results.


## Species composition of a shark fin market

Let's load our `tidyverse` libraries so we can analyze the data set^[Note: any time you start a new `Rsession` this is the code chunk you will need to re-run!].

```{r}

# load libraries
library(tidyverse)
library(ggthemes)
library(knitr)

```

The first thing we need to do is read in our blast results.

```{r}

fins <- read_delim("data/fin_blastresults.txt", delim = "\t")

```

Let's take a look at the information in the data frame.

```{r}

head(fins)

```

The first column gives us the squence ID and the last column the species ID. You should recognize the information from the column `evalue` and `pident` from your blast searches.

Currently, the species information is in a single column. Let's turn that into two, one for the genus and one for the species name.

```{r}

fins <- fins %>%
  separate(sp, into = c("genus", "species"), sep = "_", remove = FALSE)

```

Now, our first question is how many different species do we have in the data set and how many samples do we have per species, let's also calculate the relative proportions per species.

```{r}

sp_counts <- fins %>%
  group_by(genus, species) %>%
  count() %>%
  ungroup %>%
  mutate(percent = (n/sum(n))*100)

```

Let's print that table and arrange it by number of samples.

```{r}

kable(
  sp_counts %>%
    arrange(desc(n)),
  digits = 2,
  caption = "Number and relative proportions of samples species."
)

```

`r msmbstyle::question(label = "ques:seven", text = "Describe your results.")`

`r msmbstyle::question(label = "ques:eight", text = "Instead of looking at species composition, determine the number of samples and relative proportions per genus and describe your results.")`

Okay, that's some what helpful but depending on how up you are on your scientific names for sharks and general shark knowledge not super informative. Let's get some additional information for our sharks.

[Fishbase](https://www.fishbase.de) is a "global biodiversity information system on fin fishes" (though information for most sharks should be able to be found here as well).

In our google drive folder for this lab, there is a spreadsheet - we are going to use fishbase to collaboratively gather some additional information. To do this, go to the [Fishbase search site](https://www.fishbase.de/search.php) to pull up the information page for each species. If the species is not on `fishbase` another good resource to get this information is the `IUCN redlist`.

`r msmbstyle::question(label = "ques:nine", text = "Pull up a fish info page on genbank for any fish and briefly describe the type of information you can find there.")`

Here is the data we are going to gather for each species:

1. Common Name (this one is just for us to have a better idea of what we are talking about).
2. Family
3. Habitat type where the species completes most of its life cycle
      + oceanic (off continental shelf in surface waters)
      + coastal (over continental shelf)
      + deep_benthic (off the shelf and close to sediment/water interface).
4. Size-at-maturity
      + small (<100 cm total length/TL)
      + large (>100cm TL)
5. IUCN conservation status
6. CITES listing
      + AppxI
      + AppxII
      + AppxIII
      + not_eval
7. Commercial fisheries
8. Gamefish (yes/no)

Once all the information for every species has been gathered, use `excel` to save as a `tab-delimited` file in your `data` folder.

Then you can read in the file to analyze (make sure to match the file path depending on what you name your file):

```{r echo=TRUE, eval=FALSE}

specie_info <- read_delim("data/shark_info.txt", delim = "\t")

```

`r msmbstyle::question(label = "ques:ten", text = "Generate a summary table for each category (excluding common name) and give a brief description of your results.")`

`r msmbstyle::question(label = "ques:eleven", text = "Discuss your results and formulate a set of conclusions on what this data set reveals about the shark fin market and trade in the Gangzhou market. This should be a full paragraph and should include a discussion of your results and what we can learn from them about the shark fin trade how it could potentially impact shark poulations, as well as, touching on limitations and advantages of using genetic barcoding as a forensic tool.")`
